### PR TITLE
[SPARK-16109][SPARKR][DOC] R more doc fixes

### PR DESCRIPTION
**Pull request by felixcheung**
## What changes were proposed in this pull request?

Found these issues while reviewing for SPARK-16090
## How was this patch tested?

roxygen2 doc gen, checked output html

### CODE REVIEW

**Comment by felixcheung** [comment]
@shivaram

**Comment by shivaram** [comment]
cc @mengxr

**Comment by shivaram** [comment]
@felixcheung Can you change the JIRA in the title to https://issues.apache.org/jira/browse/SPARK-16109 ? I created a sub-task for statfunctions

File: R/pkg/R/stats.R [hunk 1]
```diff
@@ -33,7 +32,7 @@ setOldClass("jobj")
 #'         of `col2`. The name of the first column will be `$col1_$col2`. Pairs that have no
 #'         occurrences will have zero as their counts.
 #'
-#' @rdname statfunctions
+#' @rdname crosstab
```
Comment (shivaram) [inline R/pkg/R/stats.R]: can we have a `@family stat_functions` here to point users to other stat functions ?

**Comment by shivaram** [comment]
Other than the comment about the `family` link the rest of the changes look good

**Comment by felixcheung** [comment]
done - @family doesn't need to be one word so added "stat functions"

**Comment by shivaram** [comment]
Thanks - rebuilt locally and the docs look good. Will merge after Jenkins passes

**Comment by shivaram** [comment]
LGTM. Merging this

**Comment by felixcheung** [comment]
Great! Thanks! Hopefully now docs are in much better shape.

**Comment by shivaram** [comment]
Yeah - thanks for all the work in cleaning this up. One thing that i was wondering is we could add some style guide checks or contribution guide rules on how to maintain documentation. We can discuss this in SPARK-16090 or in a new JIRA

**Comment by felixcheung** [comment]
I have been thinking about the same thing - problem is people generally do not read contribution guide :) if there is a way to get automatic style check it would be good.

**Comment by shivaram** [comment]
Irrespective of people reading the guide, it'll at least be useful to point out what is the expected behavior in a code review etc. But yeah automatic style checks would be really cool.

**Pull request closed without merging**